src/three/TilesRenderer.d.ts

```diff
@@ -13,9 +13,11 @@ export class TilesRenderer extends TilesRendererBase {
 
 	group : TilesGroup;
 
-	getBoundsTransform( target: Matrix4 ) : Boolean;
-
 	getBounds( box : Box3 ) : Boolean;
+	getOrientedBounds( box, matrix ) : Boolean;
+	getBoundingSphere( sphere ) : Boolean;
+
+	raycast( raycaster, intersects ) : Boolean;
```

Comment:
This function is not documented and I didn't intend of it to be public. You can use the raycaster on the group if you need a raycasting API - so why don't we remove this function from the definition.

I also notice that `getBoundingSphere` isn't documented but it's probably worth adding it to the README.